src/lib/connections/bridge

```diff
@@ -25,6 +25,16 @@ bridge_up() {
     done
 
     bring_interface_up "$Interface"
+
+    if [[ ! -z "$MACAddressOf" ]]; then
```

Comment:
The `! - z` part and the quotes are omitted for these tests in the rest of netctl. This is not really documented anywhere, but it would be nice to keep things consistent.

Reply:
If I understand correctly, I think I took care of this.
Changing it from:
```if [[ ! -z "$MACAddressOf" ]]; then```
To:
```if [[ $MACAddressOf ]]; then```